byoeb-v1/byoeb/tests/integration/utils/test_embedding_cache.py

```
@@ -29,18 +15,20 @@ def get_cache_hit(resp: Any) -> bool:
 def has_devanagari(text: str) -> bool:
     return any("\u0900" <= ch <= "\u097f" for ch in text)
 
-async def run_queries(queries: List[str], features: Set[Literal["audio", "history"]] = set()) -> AsyncIterator[Any]:
-    async with Client(MCP_URL) as client:
+async def run_queries(base_url: AnyHttpUrl, access_token: str, queries: List[str], features: Set[Literal["audio", "history"]] = set()) -> AsyncIterator[Any]:
```

Ruff (0.15.10)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)